test/integration/kpatch-test

```diff
@@ -43,10 +43,9 @@ shopt -s nullglob
 # shellcheck disable=SC2046
 SCRIPTDIR=$(readlink -f $(dirname $(type -p "$0")))
 ROOTDIR=$(readlink -f "$SCRIPTDIR/../..")
-# TODO: option to use system-installed binaries instead
-KPATCH="sudo $ROOTDIR/kpatch/kpatch"
+KPATCH="${KPATCH:-sudo $ROOTDIR/kpatch/kpatch}"
```

Comment:
Looks good, a minor nit-pick should the `sudo` prefix stay,  `KPATCH="sudo ${KPATCH:-sudo $ROOTDIR/kpatch/kpatch}"`.  I am not sure if the `TODO` is still in plan and it should be retained.